format/base.go

```diff
@@ -151,37 +151,37 @@ func escape(text string) string {
 
 	// Escape the end of the string after the last URL if there's anything left
 	if len(b) > cursor {
-		builder.Write(escapeRaw(b[cursor:]))
+		builder.Write(EscapeRaw(b[cursor:]))
 	}
 
 	return builder.String()
 }
 
-func escapeRaw(segment []byte) []byte {
+func EscapeRaw(segment []byte) []byte {
 	return specialCharacterRegex.ReplaceAll(segment, []byte("\\$1"))
 }
 
 // plainText converts a markdown string to the plain text that appears in the
 // rendered output.
-func plainText(text string) string {
+func PlainText(text string) string {
 	md := blackfriday.New(blackfriday.WithExtensions(blackfriday.CommonExtensions))
 	node := md.Parse([]byte(text))
 
 	var builder strings.Builder
-	plainTextInner(node, &builder)
+	PlainTextInner(node, &builder)
 
 	return builder.String()
 }
 
-func plainTextInner(node *blackfriday.Node, builder *strings.Builder) {
+func PlainTextInner(node *blackfriday.Node, builder *strings.Builder) {
```

Comment:
Same as `EscapeRaw`. Let's leave this unexported unless there's a specific reason you think it should be exposed